src/room/Room.ts

```diff
@@ -1050,9 +1051,16 @@ class Room extends (EventEmitter as new () => TypedEmitter<RoomEventCallbacks>)
 
     this.engine.client.sendSyncState({
       answer: toProtoSessionDescription({
-        sdp: previousSdp.sdp,
-        type: previousSdp.type,
+        sdp: previousAnswer.sdp,
+        type: previousAnswer.type,
       }),
+      offer:
+        previousOffer !== null
```

Comment:
default value is undefined, which is diff from null. using "trueness" operator would be fine in this case.

```suggestion
        previousOffer ? toProtoSessionDescription
```